installing/installing_oci/installing-oci-agent-based-installer.adoc

```diff
@@ -21,6 +23,22 @@ include::modules/installing-oci-about-agent-based-installer.adoc[leveloffset=+1]
 * link:https://docs.oracle.com/en-us/iaas/Content/Block/Concepts/blockvolumeperformance.htm#vpus[Volume Performance Units (Oracle documentation)]
 * link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-agent-about-instance-configurations.htm[Instance Sizing Recommendations for {product-title} on {oci} Nodes (Oracle documentation)]
 
+[id="abi-oci-process-checklist_{context}"]
+== Installation process workflow
+
+The following workflow describes a high level outline for the process of installing an {product-title} cluster on {oci} using the Agent-based Installer:
```

Comment:
```suggestion
The following workflow describes a high-level outline for the process of installing an {product-title} cluster on {oci} using the Agent-based Installer:
```

Reply:
Should this intro explain that the list is showing where each step is performed with the items in parentheses?

It would be out of scope for this PR, but you might even think about doing this as a table since you have two columns of info here:

| Step      | Where       |
| ------------- | ------------- |
| 1. Create OCI resources and services | Oracle |
| 2. Disconnected environments: Prepare a web server that is accessible by OCI instances | Red Hat |
| 3. Prepare configuration files for the Agent-based Installer | Red Hat |

(etc)

Reply:
That's a solid idea actually, I will consider that. I don't want to set up a table while merging is the priority, but I think I am going to refine this doc later and can make this a table at that point.